samples/snippets/src/main/java/pubsublite/CreateSubscriptionExample.java

```diff
@@ -41,24 +42,39 @@ public static void main(String... args) throws Exception {
     String topicId = "your-topic-id";
     String subscriptionId = "your-subscription-id";
     long projectNumber = Long.parseLong("123456789");
+    boolean regional = false;
```

Comment:
Can you include more context or a link to documentation that describes what effect this has?

Reply:
Done.